api-tests/management/mysql_test.go

```diff
@@ -595,6 +595,7 @@ func TestAddMySQL(t *testing.T) {
 		assert.Nil(t, addMySQLOK)
 	})
 
+	// according to credentials-source parameter, passing empty username is allowed
```

Comment:
it should be allowed only if `credentials source` is passed. In other cases behaviour should be the same.

Reply:
the same in all cases below

Reply:
Changed.